Source/Core/InputCommon/ControllerEmu/ControlGroup/IMUGyroscope.h

```diff
@@ -23,7 +23,7 @@ class IMUGyroscope : public ControlGroup
 
   StateData GetRawState() const;
   // Also updates the state by default
```

Comment:
Is this comment still correct after this change? Since `update` was apparently never set (so it was always `true`) this should probably be something like "Also updates the state" (whatever this means.)